app/components/ui/files/registration-modal/component.js

```diff
@@ -108,9 +110,15 @@ export default Ember.Component.extend({
     },
 
     actions: {
+
+        updateDev(value) {
+            // Called if the `use dev` checkbox is clicked
+            this.set('useDev', value)
```

Comment:
This is really minor, @ThomasThelen, but I favor ending lines with a semicolon as a rule that I even try enforcing with my editor.